apps/main-app/src/components/paralogy/paralogyFilteredTable.js

```diff
@@ -0,0 +1,215 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Collapse } from 'reactstrap';
+import StringencySelection from '../orthology/stringencySelection';
+import { isBest } from '../orthology/orthologyTable';
+import { getOrthologSpeciesName } from '../orthology/utils';
+import HorizontalScroll from '../horizontalScroll';
+import LoadingSpinner from '../loadingSpinner';
+import NoData from '../noData';
+import ControlsContainer from '../controlsContainer';
+import { STRINGENCY_HIGH } from '../orthology/constants';
+import {
+  compareAlphabeticalCaseInsensitive,
+  orthologyMeetsStringency
+} from '../../lib/utils';
+import HelpPopup from '../helpPopup';
+import OrthologyFilterHelp from '../orthology/orthologyFilterHelp';
+import useResettableState from '../../hooks/useResettableState';
+import useGeneParalogy from '../../hooks/useGeneParalogy';
+import ParalogyTable from './paralogyTable';
+
+const ParalogyFilteredTable = ({geneId}) => {
+  const [
+    filterScoreGreaterThan,
+    setFilterScoreGreaterThan,
+    resetFilterScoreGreaterThan
+  ] = useResettableState(0);
+  const [
+    filterMethod,
+    setFilterMethod,
+    resetFilterMethod
+  ] = useResettableState(null);
+  const [
+    filterBest,
+    setFilterBest,
+    resetFilterBest
+  ] = useResettableState(false);
+  const [
+    filterReverseBest,
+    setFilterReverseBest,
+    resetFilterReverseBest
+  ] = useResettableState(false);
+  const [
+    filterSpecies,
+    setFilterSpecies,
+    resetFilterSpecies
+  ] = useResettableState(null);
+  const [
+    stringencyLevel,
+    setStringencyLevel,
+    resetStringencyLevel
+  ] = useResettableState(STRINGENCY_HIGH);
+  const [showFilterPanel, setShowFilterPanel] = useState(false);
+
+  const { data, isLoading } = useGeneParalogy(geneId);
+
+  const filterCallback = (dat) => {
+    const meetMethodFilter = filterMethod ?
+      dat.predictionMethodsMatched.indexOf(filterMethod) > -1 :
+      true;
+    return (
+      meetMethodFilter &&
+      dat.predictionMethodsMatched.length > filterScoreGreaterThan &&
+      (filterBest ? isBest(dat.best) : true) &&
+      (filterReverseBest ? isBest(dat.bestReverse) : true) &&
+      (filterSpecies ? getOrthologSpeciesName(dat) === filterSpecies : true) &&
```

Comment:
I don't think we need the filterSpecies...

Reply:
Removed